doc/data-files-description.md

```diff
@@ -53,10 +53,14 @@ This document contains information about all data files associated with this pro
 |`pbta-isoform-counts-rsem-expected_count.stranded.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/kfdrc_RNAseq_workflow.cwl) |Gene expression - RSEM expected counts for stranded samples (transcript-level)
 |`pbta-isoform-expression-rsem-tpm.polya.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for poly-A samples (transcript-level)
 |`pbta-isoform-expression-rsem-tpm.stranded.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for stranded samples (transcript-level)
-|`pbta-snv-consensus-mutation-tmb.tsv` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Tumor mutation burden statistics calculated from consensus SNV, using Strelka2 counts and BED window sizes
+|`pbta-snv-consensus-mutation-tmb-all.tsv` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Tumor mutation burden statistics calculated from consensus SNV, using Strelka2 counts and BED window sizes
```

Comment:
@cansavvy can you reply to this comment with what you would say for the file descriptions for the all TMB and coding TMB files please? I will also port that information over to `doc/data-formats.md`.

Reply:
For all: ```Tumor mutation burden statistics calculated from Strelka2 and Mutect2 SNV consensus, and the intersection of Strelka2 and Mutect2 BED windows sizes.```
For coding: ```Coding only tumor mutation burden statistics calculated from the number of exonic Strelka2, Mutect2, and Lancet consensus SNVs and size of the intersection of all three callers' BED windows and the Gencode v27 exome. ```